extensions/amp-geo/amp-geo.md

```diff
@@ -131,6 +131,24 @@ The game is called <span class='football'></span>!
 </div>
 ```
 
+### Render Blocking
+
+By default, the `amp-geo` component is not render blocking. That is, the page will load and elements will render even if `amp-geo` has not yet loaded and executed. If it's important that certain elements are never rendered in a specific geography, use the `amp-geo-pending` class to provide selective render blocking. This is implemented by the publisher by adding `amp-geo-pending` to the `<body>` element. When it loads, `amp-geo` removes this class at the same time as it adds the `amp-iso-country...` and `amp-geo-group-...` classes.
```

Comment:
"When it loads" - clarify what "it" is.  The page, the body, the amp-geo-pending class, the amp-geo component?

Reply:
I clarified that it's the amp-geo script that's loading.